contents/handbook/growth/marketing/index.md

```diff
@@ -9,17 +9,27 @@ showTitle: true
 Philosophy: Be kind, concise and direct.
 
 
+## Positioning statement
+
+We use a positioning statement based on a template by Geoffrey Moore, which enables us to establish a position relative to the rest of the market. Our positioning statement helps us think about how we can continue to improve and evolve PostHog ahead of alternative tools.
+
+Our positioning statement is:
+
+_For product managers and software engineers who need to understand their users, PostHog is the only open source analytics platform which enables you to discover insights and run experiments._ 
+
+_Unlike traditional analytics tools, PostHog offers self-hosted deployments so that data doesn't need to be shared with anyone -- not even PostHog._ 
+
 ## Product positioning
 
-An **open source product analytics platform** addresses the lack of choice and control amongst disconnected analytics solutions by offering a **unified platform** with **control** over hosting, pricing, source, data, privacy and security.
+An **open source product analytics platform**, PostHog addresses the lack of choice and control amongst disconnected analytics solutions by offering a **unified platform** with **control** over hosting, pricing, source, data, privacy and security.
 
-**Free PostHog** is positioned to solve product analytics problems for small non-enterprise teams.
+**PostHog Open Source** is positioned to service hobbyists and very small non-enterprise teams who may want to retain control of their data. 
 
-**Enterprise PostHog** is positioned to solve product analytics problems for larger teams and enterprises.
+**PostHog Free** is positioned to service small non-enterprise teams who need to retain control of their data.
```

Comment:
We're about to rejig these - PostHog Open source and PostHog free are now _in practice_ merged. 


* We're currently  updating the docs to reflect this change.
* This means they're now used to service 1-3 users in teams that range in size from hobbyists through to small deployments in enterprises. The goal is no matter which size of company and how much scale you have, that you can go into production with our open source product.

Reply:
Ah, wasn't aware this was so close to merging. 

Will we keep the two distinct, or do we plan to merge them into a single product. The simplest offering/naming would seem to be: Open-Source, Scale and Cloud. 

Would be good to understand this so I can update the pricing and features copy when needed too.

Reply:
I didn't think it'd be so fast either, but Team Deployments and Infrastructure is very good :)

They'll definitely like one product, with different config options in the way it gets deployed I imagine.